app/models/project.rb

```diff
@@ -27,8 +27,8 @@ class Project < ActiveRecord::Base
   settings number_of_shards: 1 do
     mapping do
       indexes :country_name, index: 'not_analyzed'
-      indexes :languages_names, index: 'not_analyzed'
-      indexes :levels_names, index: 'not_analyzed'
+      indexes :languages_name, index: 'not_analyzed'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.